### ISSUE TITLE
Install from zip fails on Mac (unmet dependancies?)

### ISSUE THREAD

**Issue by rabennett** [EarthObservation/rvt-qgis#5]
Install from zip fails with attached message: (plugin manager states install successful but no toolbox in Raster menu)
![Screen Shot 2021-01-06 at 15 44 49-2](https://user-images.githubusercontent.com/2912586/103794854-2641b680-503d-11eb-94d5-06f2f0bae96f.png)

QGIS spec:
![Screen Shot 2021-01-06 at 16 22 58](https://user-images.githubusercontent.com/2912586/103794883-2f328800-503d-11eb-8430-2bf920593964.png)

**Comment by zm8597** [comment]
You don't have GDAL. Install GDAL (https://www.kyngchaos.com/software/qgis/, https://www.kyngchaos.com/software/frameworks/).

**Comment by zm8597** [comment]
I created Ubuntu (in VirtualBox) and found out that QGIS uses wrong path to python gdal library. I resolved this issue with adding the right path (path to python gdal directory) to QGIS enviroment variables. In terminal write "python3 -m pip install gdal" (to make sure python gdal library is installed) the terminal will probably return something like this: "Requirement already satisfied: gdal in /usr/lib/python3/dist-packages (3.0.4)". Copy this path ("/usr/lib/python3/dist-packages"). Open QGIS go to Settings->Options->System, scroll down to the Environment section and check "Use custom variables" checkbox. Then click plus button (add) and under Apply drop down select Prepend, under Variable write "PYTHONPATH" and under Value paste the path you copied. Click OK and restart QGIS, after that everything should work fine.
![image](https://user-images.githubusercontent.com/67868248/104815915-fbfbb000-5817-11eb-9a3a-5b662cc802ac.png)
